cmd/soroban-cli/src/commands/contract/mod.rs

```diff
@@ -91,6 +92,9 @@ pub enum Cmd {
     ///
     /// If no keys are specificed the contract itself is restored.
     Restore(restore::Cmd),
+
+    /// Get the wasm hash of a deployed contract
+    WasmHash(wasm_hash::Cmd),
```

Comment:
Can you register the wasm-hash command on the `contracts info` command, rather than on the `contracts` command?